<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Wybierz atrybuty do skopiowania</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zaznacz_wszystko">
     <property name="text">
      <string>Zaznacz wszystko</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="odznacz_wszystko">
     <property name="text">
      <string>Odznacz wszystko</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="potwierdz_wybor">
     <property name="text">
      <string>Potwierdź wybór</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Wybierz obiekt źródłowy, a następnie obiekty docelowe.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="wybierz_source">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Kopiuj atrybuty</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="wybierz_destination">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Wklej atrybuty</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cofnij_destination">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cofnij</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
